### ISSUE TITLE
Penalized regression modelling for cph

### ISSUE THREAD

**Issue by sumprain** [harrelfe/rms#9]
Respected Sir,
Will it be possible to run penalised regression for models made by cph, in the same lines as is possible with pentrace for ols/lrm models?
Thanks for the excellent packages rms and Hmisc

**Comment by harrelfe** [comment]
No.  Therneau in the coxph function in the survival package provides the 
building blocks for that but I haven't put them together for rms.

Frank


On 07/13/2015 06:49 AM, Suman Kumar Pramanik wrote:
>
> Respected Sir,
> Will it be possible to run penalised regression for models made by 
> cph, in the same lines as is possible with pentrace for ols/lrm models?
> Thanks for the excellent packages rms and Hmisc
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/harrelfe/rms/issues/9>.
>

-- 
------------------------------------------------------------------------
Frank E Harrell Jr 	Professor and Chairman 	School of Medicine

	Department of *Biostatistics* 	*Vanderbilt University*
